src/cassandra/uuid.rs

```diff
@@ -104,6 +104,56 @@ impl Uuid {
     }
 }
 
+impl From<uuid::Uuid> for Uuid {
+    fn from(id: uuid::Uuid) -> Uuid {
+        // implementation taken from serializetion.hpp encode_uuid()
+        let input = id.as_bytes();
+
+        let mut time_and_version = 0u64;
+        time_and_version |= input[3] as u64;
+        time_and_version |= (input[2] as u64) << 8;
+        time_and_version |= (input[1] as u64) << 16;
+        time_and_version |= (input[0] as u64) << 24;
+
+        time_and_version |= (input[5] as u64) << 32;
+        time_and_version |= (input[4] as u64) << 40;
+
+        time_and_version |= (input[7] as u64) << 48;
+        time_and_version |= (input[6] as u64) << 56;
+
+        let mut clock_seq_and_node = 0u64;
+        for i in 0..8 {
+            clock_seq_and_node |= (input[15 - i] as u64) << (8 * i);
+        }
+        Uuid(_Uuid {
+            time_and_version,
+            clock_seq_and_node,
+        })
+    }
+}
+
+impl Into<uuid::Uuid> for Uuid {
```

Comment:
You shouldn't implement `Into` directly; instead please provide
```rust
impl From<Uuid> for uuid::Uuid
```
See https://doc.rust-lang.org/std/convert/trait.Into.html#implementing-into-for-conversions-to-external-types-in-old-versions-of-rust

Reply:
I used Into because uuid::Uuid was an external type. Looks like they changed things so that it works even for external types. I will try that.

Reply:
Thanks - yes, it changed to allow this sort of situation.